3.2/root/usr/share/container-scripts/mongodb/README.md

```diff
@@ -88,3 +91,38 @@ the values stored in the variables and the actual passwords. Whenever a database
 container starts it will reset the passwords to the values stored in the
 environment variables.
 
+
+Extending image
+---------------------------------
+This image can be extended using [source-to-image](https://github.com/openshift/source-to-image).
+
+For example to build customized MongoDB database image `my-mongodb-centos7` with configuration in `~/image-configuration/` run:
+
+```
+$ s2i build ~/image-configuration/ centos/mongodb-32-centos7 my-mongodb-centos7
+```
+
+The directory passed to `s2i build` can contain these directories:
```

Comment:
I'd be more specific here: "...should contain one or more of the following directories:"